### ISSUE TITLE
'@kwsites/file-exists' does not provide an export named 'FOLDER'

### ISSUE THREAD

**Issue by LukyVj** [steveukx/git-js#763]
Hello, I'm double posting about an issue on [`file-exists`](https://github.com/kwsites/file-exists/issues/10)

---- 

I get this error while using your package:
```
 import { exists, FOLDER } from "@kwsites/file-exists";
 ^^^^^^
 SyntaxError: The requested module '@kwsites/file-exists' does not provide an export named 'FOLDER'
```
For more context, I use simple-git on Heroku, and the logs returns this error.

**Comment by steveukx** [comment]
Hello, please can you share the code you're using to import `simple-git` ? Are you using TypeScript / CommonJS or ESM?

**Comment by LukyVj** [comment]
Hello, 

I'm using it within an ESM file. 

I've ended up using an other strategy to achieve what I wanted, but I guess it can still help others if this issue is fixed. 

Thanks.

**Comment by steveukx** [comment]
There is an integration test https://github.com/steveukx/git-js/tree/main/packages/test-es-module-consumer that documents using `simple-git` with ESM, so I'm not sure why you were seeing that error.

In order to take any action on fixing something, please share your code so I can investigate the problem.

**Comment by LukyVj** [comment]
Really weird, indeed. The start of my file is exactly like the one in your example, so I'm not really sure. 

I'm going to share with you the related part of my file: 
```js
import "dotenv/config";
import core from "puppeteer-core";
import fs from "fs";

import simpleGit, { CleanOptions } from "simple-git";
simpleGit().clean(CleanOptions.FORCE);

const today = new Date();
const git = simpleGit();
const isDev = process.env.NODE_ENV === "development";

const remote1 = `https://${process.env.GH_USER}:${process.env.GH_TOKEN}@${process.env.GH_REPO}`;
```

**Comment by bcanseco** [comment]
Not sure the actual root cause, but I had this same error message and fixed it by upgrading from node@14.3.0 to node@16.11.1.
